src/lib/runtime_config/runtime_config.ml

```diff
@@ -1017,6 +1143,101 @@ let combine t1 t2 =
   ; epoch_data = opt_fallthrough ~default:t1.epoch_data t2.epoch_data
   }
 
+let gen =
+  let open Quickcheck.Generator.Let_syntax in
+  let%map daemon = Daemon.gen
+  and genesis = Genesis.gen
+  and proof = Proof_keys.gen
+  and ledger = Ledger.gen
+  and epoch_data = Epoch_data.gen in
+  { daemon = Some daemon
+  ; genesis = Some genesis
+  ; proof = Some proof
+  ; ledger = Some ledger
+  ; epoch_data = Some epoch_data
+  }
+
+let ledger_accounts (ledger : Mina_base.Ledger.Any_ledger.witness) =
+  Mina_base.Ledger.Any_ledger.M.foldi ~init:[]
+    ~f:(fun _ accum act -> act :: accum)
```

Comment:
This will reverse the list of accounts, which will result in a different merkle root when you rebuild the ledger. I believe there already exists an `accounts` function on the ledger interface which will do what you want.

Reply:
Done in https://github.com/MinaProtocol/mina/pull/14266/commits/20789695c1aa430f7f6c30cd0dea25fcc38218e6.